Samples/word-citation-management/README.md

```diff
@@ -0,0 +1,140 @@
+
+The sample also depends on [@orcid/bibtexParseJs](https://github.com/ORCID/bibtexParseJs#readme), a JavaScript library for parsing the .bib file.
+
+![Citation management add-in task pane.](./assets/word-citation-management.png)
+
+## Applies to
+
+- Word on Windows
+- Word on Mac
+- Word on the web
+
+## Prerequisites
+
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+- [@orcid/bibtexParseJs](https://github.com/ORCID/bibtexParseJs#readme).
+
+## Solution
+
+| Solution | Authors |
+|----------|-----------|
+| How to manage citations in a Word document | Microsoft |
+
+## Version history
+
+| Version  | Date | Comments |
+|----------|------|----------|
+| 1.0 | 11-15-2023 | Initial release |
+
+## Run the sample
+
+1. Clone or download this repo.
+
+1. Go to the **Samples\word-citation-management** folder via the command line.
+
+1. Run `npm install`.
+
+1. Run `npm run build`.
+
+1. Start the local web server and sideload your add-in.
+
+    - To test your add-in in Word, run the following command in the root directory of your project. This starts the local web server (if it's not already running) and opens Word with your add-in loaded.
+
+      - Windows: `npm start`
+      - Mac: `npm run dev-server`
+
+    - To test your add-in in Word on the web, run the following command in the root directory of the `word-citation-management` project. When you run this command, the local web server starts. Replace "{url}" with the URL of a Word document on your OneDrive or a SharePoint library to which you have permissions.
+
+      ```command line
+      npm run start:web -- --document {url}
+      ```
+
+      The following are examples.
+
+      - `npm run start:web -- --document https://contoso.sharepoint.com/:t:/g/EZGxP7ksiE5DuxvY638G798BpuhwluxCMfF1WZQj3VYhYQ?e=F4QM1R`
+      - `npm run start:web -- --document https://1drv.ms/x/s!jkcH7spkM4EGgcZUgqthk4IK3NOypVw?e=Z6G1qp`
+      - `npm run start:web -- --document https://contoso-my.sharepoint-df.com/:t:/p/user/EQda453DNTpFnl1bFPhOVR0BwlrzetbXvnaRYii2lDr_oQ?e=RSccmNP`
+
+      > NOTE: If you're developing on a Mac, enclose the {url} in single quotation marks. Don't do this on Windows.
+
+      If your add-in doesn't sideload in the document, manually sideload it by following the instructions in [Manually sideload add-ins to Office on the web](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing).
```

Comment:
Manually uploading the manifest to Word on the web resulted in this error:

![image](https://github.com/OfficeDev/Office-Add-in-samples/assets/38896772/122fe9bf-65fb-4a66-88ea-59e7b548025e)